spec/stub_classes.rb

```diff
@@ -16,6 +16,23 @@ def initialize(alerts = [])
     end
   end
 
+  class StubObjectSummary
```

Comment:
This stub class is getting more complicated... I've had good luck with Mocha in the past:
https://github.com/freerange/mocha#quick-start